tools/nightly-unit-tests/generate_bitrise_yml.py

```diff
@@ -118,6 +118,11 @@ def generate_bitrise_yml(test_plan: TestPlan, dry_run: bool):
 
     print('\n⚙️ Saving `bitrise.yml`...')
     bitrise_yml.write(path='bitrise.yml')
+
+    print('\n⚙️ `bitrise.yml` content:')
+    saved_bitrise_yml = open('bitrise.yml', 'r')
+    print(saved_bitrise_yml.read())
+
```

Comment:
q/ Do we need it? AFAIR generated `yml` is deployed as build artifact and available after job completion.

Reply:
ah, you are right, it is there. I didn't know that, Just removed this part.